components/ProfilePage/StyledProfileComponents.tsx

```diff
@@ -34,6 +34,18 @@ export const UserIcon = styled(Image).attrs(props => ({
   margin-right: 2rem;
 `
 
+export const UserIconSmall = styled(Image).attrs(props => ({
+  alt: "",
```

Comment:
I think this should be filled in with a translatable string? Or maybe I'm not understanding how the `.attrs` works...

Reply:
I was lazily following a pre-existing pattern of leaving it unfilled in when clearly it ought to be; fixed